rn-cli.config.js

```diff
@@ -15,7 +15,7 @@ const sharedBlacklist = [
   /website\/node_modules\/.*/,
   /heapCapture\/bundle\.js/,
   /\.build[/\\].*/,
-  /ios[/\\]build[/\\]Build[/\\].*/,
+  /ios[/\\]build[/\\].*/,
```

Comment:
This change is related to me starting upgrade to RN 0.59. It seems it has changes the ios build folder.